sympy/polys/factortools.py

```diff
@@ -169,7 +169,7 @@ def dup_zz_mignotte_bound(f, K):
     delta2 = _ceil(delta / 2)
 
     # euclidean-norm
-    eucl_norm = K.sqrt( sum( [cf**2 for cf in f] ) )
+    eucl_norm = _sqrt( sum( [cf**2 for cf in f] ) )
```

Comment:
This will change the type of `eucl_norm` that is an element of `K`. What is the purpose of the change?

Reply:
Alkis asked me to change this, because he founds that now it is passing an test. Although for me it stacks again in an infinite loop on `test_Poly_lift` file. Now i will try to find the periodicity you mentioned above